config/polyfills.js

```diff
@@ -20,3 +20,7 @@ Object.assign = require('object-assign');
 if (process.env.NODE_ENV === 'test') {
   require('raf').polyfill(global);
 }
+
+// Needed by async/await and Babel 7
+require("babel-polyfill");
```

Comment:
shouldn't it be `@babel/polyfill` ? this only works thanks to the babel-core@7.0.0-bridge